src/core/Akka.Cluster/Cluster.cs

```diff
@@ -282,7 +282,7 @@ public Task JoinAsync(Address address, CancellationToken token = default)
 
             var completion = new TaskCompletionSource<NotUsed>(TaskCreationOptions.RunContinuationsAsynchronously);
 
-            if (token != default)
+            if (token != CancellationToken.None)
```

Comment:
Does `CancellationToken.None` equals to `default`? Because the default method argument value is `default`.

Reply:
Yes, this is a C# linter suggestion - so I'd assume they're equal

Reply:
Reverted this